SpatialGDK/Source/Private/Interop/SpatialSender.cpp

```diff
@@ -407,14 +409,14 @@ void USpatialSender::QueueOutgoingUpdate(USpatialActorChannel* DependentChannel,
 		AnotherHandleToUnresolved.Add(Handle, Unresolved);
 
 		// Following up on the previous log: listing the unresolved objects
-		UE_LOG(LogTemp, Log, TEXT("%s"), *UnresolvedObject->GetName());
+		UE_LOG(LogSpatialSender, Log, TEXT("%s"), *UnresolvedObject->GetName());
```

Comment:
Lets add a `- %s` or `    %s` to makle it obvious these logs are a part of the previous.